content/systems/filecoin_vm/actor/_index.md

```diff
@@ -2,15 +2,67 @@
 title: Actor
 weight: 1
 dashboardWeight: 2
-dashboardState: incorrect
+dashboardState: incomplete
 dashboardAudit: 0
 dashboardTests: 0
 ---
 
 # VM Actor Interface
 ---
 
+As mentioned above, Actors are the Filecoin equivalent of smart contracts in the Ethereum Virtual Machine. As such, Actors are very central components of the system. Any change to the current state of the Filecoin blockchain has to be triggered through an actor method invocation.
 
-{{<embed src="actor.id" lang="go" >}}
+This sub-section describes the _interface_ between Actors and the Filecoin Virtual Machine. This means that most of what is described below does not strictly belong to the VM. Instead it is logic that sits on the interface between the VM and Actors logic.
 
-{{<embed src="actor.go" lang="go" >}}
+There are eleven (11) types of _builtin_ Actors in total, not all of which interact with the VM. Some Actors do not invoke changes to the StateTree of the blockchain and as such do not need to have an interface to the VM. We discuss the details of all System Actors later on in the System Actors subsection.
```

Comment:
The Code ID *is* a CID (Content Identifier), confusing I know. I'd drop the *code* term in Actors and just talk about IDs that are backed by CIDs.